hpke-double-ext.md

```diff
@@ -0,0 +1,172 @@
+# Introduction
+
+In some applications, where a party serves as a relay for batches of
```

Comment:
Maybe add the scrambledb motivation and use somewhere here as well.

Reply:
I wanted to leave the description of the extension as if it stood on its own. But, of course, we only want to use it in the ScrambleDB context. Do you mean using ScrambleDB as one possible example where this would be useful in the introduction, or should I use the ScrambleDB terminology (e.g. "Converter" instead of "Relay") in the rest of the extension as well so it doesn't appear more generic than it really is?

Reply:
You motivate the double HPKE version here with "some applications". So it would be nice to know which applications. And ScrambleDB is the one we have.
So mentioning that explicitely would make sense. You don't need to use any ScrambleDB teminology for that here.

Reply:
You're right, I've made ScrambleDB explicit as one application of this, which also better motivates why you wouldn't use decryption & re-encryption instead to achieve the same effect.